xml/System.Windows.Media/Drawing.xml

```diff
@@ -29,7 +29,7 @@
 ## Remarks  
  <xref:System.Windows.Media.Drawing> objects are light-weight objects that enable you to add geometric shapes, images, text, and media to an application. <xref:System.Windows.Media.Drawing> objects are considered light-weight because they do not provide support for [Layout](~/docs/framework/wpf/advanced/layout.md), [Input Overview](~/docs/framework/wpf/advanced/input-overview.md), and focus. Because of their performance benefits, drawings are ideal for backgrounds and clip art. You also use drawings when programming at the <xref:System.Windows.Media.Visual> level.  
   
- Because they inherit from the <xref:System.Windows.Freezable> class, <xref:System.Windows.Media.Drawing> objects  provide additional features that make them useful for describing clip art and backgrounds: they can be declared as , shared among multiple objects, made read-only, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see the [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ Because they inherit from the <xref:System.Windows.Freezable> class, <xref:System.Windows.Media.Drawing> objects  provide additional features that make them useful for describing clip art and backgrounds: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see the [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 Because they inherit from the <xref:System.Windows.Freezable> class, <xref:System.Windows.Media.Drawing> objects  provide additional features that make them useful for describing clip art and backgrounds: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only, and made thread-safe. For more information about the different features provided by <xref:System.Windows.Freezable> objects, see the [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```